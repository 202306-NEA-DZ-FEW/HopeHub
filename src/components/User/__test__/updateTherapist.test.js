import renderer from "react-test-renderer";

import TherapistProfile from "../updateTherapist";

const user = {
    // Populate with necessary user data for testing
    name: "John Doe",
    birthDate: "1990-01-01",
    email: "johndoe@example.com",
    phoneNumber: "1234567890",
    // Add other user properties as needed
};

it("renders correctly", () => {
    const tree = renderer.create(<TherapistProfile user={user} />).toJSON();
    expect(tree).toMatchSnapshot();
});
