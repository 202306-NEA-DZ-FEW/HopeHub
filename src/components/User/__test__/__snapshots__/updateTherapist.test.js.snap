// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-NeutralWhite dark:bg-Dark_Accent"
  dir="ltr"
>
  <div
    className="navbar h-8 sticky top-0 z-10 dark:bg-NeutralBlack dark:backdrop-blur-lg dark:bg-opacity-30 bg-white backdrop-filter backdrop-blur-lg bg-opacity-30 border-b-slate-400"
  >
    <div
      className="navbar-start w-auto"
    >
      <div
        className="dropdown"
      >
        <label
          className="btn btn-ghost md:pl-6 lg:hidden"
          tabIndex={0}
        >
          <svg
            className="h-5 w-5"
            fill="none"
            stroke="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M4 6h16M4 12h8m-8 6h16"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
            />
          </svg>
        </label>
        <ul
          className="menu menu-sm dropdown-content -ml-2 px-3 shadow w-screen mt-2 bg-Primary"
          tabIndex={0}
        >
          <li
            className="py-1 text-NeutralBlack font-semibold dark:text-NeutralWhite"
          >
            <a
              className="text-lg"
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Home
            </a>
          </li>
          <li
            className="py-1 text-NeutralBlack font-semibold dark:text-NeutralWhite"
          >
            <a
              className="text-lg"
              href="/blogs"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Blogs
            </a>
          </li>
          <li
            className="py-1 text-NeutralBlack font-semibold dark:text-NeutralWhite"
          >
            <a
              className="text-lg"
              href="/about"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              About
            </a>
          </li>
          <li
            className="py-1 text-NeutralBlack font-semibold dark:text-NeutralWhite"
          >
            <a
              className="text-lg"
              href="/contact"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Contact
            </a>
          </li>
          <div
            className="border-t-2 border-NeutralWhite pb-4"
          />
          <li>
            <button
              className="button-container w-32 mx-2 mb-1 h-9 md:mr-8 md:w-40 bg-Accent hover-bg-Primary rounded-md"
            >
              <a
                className="text-lg tracking-wider text-NeutralBlack font-semibold font-poppins"
                href="/Auth"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "textTransform": "capitalize",
                  }
                }
              >
                Log In
              </a>
            </button>
          </li>
        </ul>
      </div>
      <div
        className="logo-container lg:px-6"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <img
            alt="Normal Logo"
            data-nimg="1"
            decoding="async"
            height={60}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=256&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=128&q=75 1x, /_next/image?url=%2Fimg.jpg&w=256&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={120}
          />
        </a>
      </div>
    </div>
    <div
      className="flex justify-between ml-auto dropdown"
    >
      <div
        className="flex items-center"
      >
        <div
          className="search-icon mr-2"
          onClick={[Function]}
        >
          <svg
            className="w-6 h-6 dark:fill-NeutralWhite"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 256 256"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M229.66,218.34l-50.07-50.06a88.11,88.11,0,1,0-11.31,11.31l50.06,50.07a8,8,0,0,0,11.32-11.32ZM40,112a72,72,0,1,1,72,72A72.08,72.08,0,0,1,40,112Z"
            />
          </svg>
        </div>
        <div
          className="search-bar"
        >
          <input
            className="input input-bordered input-sm w-full"
            onChange={[Function]}
            placeholder="Type here"
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        className="navbar-center mr-4 hidden lg:flex"
      >
        <ul
          className="menu menu-horizontal"
        >
          <li
            className="text-Accent font-semibold font-poppins text-base tracking-wider text-NeutralBlack font-semibold dark:text-NeutralWhite"
          >
            <a
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Home
            </a>
          </li>
          <li
            className="text-Accent font-semibold font-poppins text-base tracking-wider text-NeutralBlack font-semibold dark:text-NeutralWhite"
          >
            <a
              href="/blogs"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Blogs
            </a>
          </li>
          <li
            className="text-Accent font-semibold font-poppins text-base tracking-wider text-NeutralBlack font-semibold dark:text-NeutralWhite"
          >
            <a
              href="/about"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              About
            </a>
          </li>
          <li
            className="text-Accent font-semibold font-poppins text-base tracking-wider text-NeutralBlack font-semibold dark:text-NeutralWhite"
          >
            <a
              href="/contact"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Contact
            </a>
          </li>
          <button
            className="button-container w-28 h-9 mx-4 md:mr-2 md:w-40 bg-Accent dark:bg-Dark_Accent dark:hover:bg-Dark_Primary hover-bg-Primary rounded-md"
          >
            <a
              className="text-base tracking-wider text-NeutralBlack dark:text-NeutralWhite font-semibold font-poppins"
              href="/Auth"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
              style={
                Object {
                  "textTransform": "capitalize",
                }
              }
            >
              Log In
            </a>
          </button>
        </ul>
      </div>
    </div>
  </div>
  <div
    className="mt-8"
  >
    <div
      className="flex justify-center font-semibold font-poppins flex-col md:flex-row  max-w-screen bg-NeutralWhite dark:bg-NeutralBlack"
    >
      <div
        className="pb-12 lg:py-16 lg:w-[60%] md:[60%] flex "
      >
        <div
          className="bg-NeutralBlack dark:bg-NeutralWhite border-2 w-80 h-80 rounded-full mx-auto flex flex-col items-center justify-center relative overflow-visible"
        >
          <svg
            className="fill-NeutralWhite dark:fill-NeutralBlack w-16 h-16 md:w-24 md:h-24 mb-5  "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
            />
          </svg>
          <input
            className="hidden"
            id="userImg"
            name="userImg"
            onChange={[Function]}
            type="file"
          />
          <label
            className="absolute -mb-[19rem] cursor-pointer "
            onClick={[Function]}
          >
            <svg
              className="text-NeutralBlack dark:text-NeutralWhite  w-12 h-12 md:w-14 md:h-14 bg-NeutralWhite dark:bg-NeutralBlack rounded-full border-2 dark:border-NeutralWhite border-NeutralBlack p-2"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 32 32"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M 13 3 C 9.144531 3 6 6.144531 6 10 C 6 12.410156 7.230469 14.550781 9.09375 15.8125 C 5.527344 17.34375 3 20.882813 3 25 L 5 25 C 5 20.570313 8.570313 17 13 17 C 15.144531 17 17.0625 17.878906 18.5 19.25 L 13.78125 23.96875 L 13.71875 24.28125 L 13.03125 27.8125 L 12.71875 29.28125 L 14.1875 28.96875 L 17.71875 28.28125 L 18.03125 28.21875 L 28.125 18.125 C 29.285156 16.964844 29.285156 15.035156 28.125 13.875 C 27.542969 13.292969 26.769531 13 26 13 C 25.246094 13 24.484375 13.285156 23.90625 13.84375 L 19.9375 17.8125 C 19.066406 16.976563 18.035156 16.292969 16.90625 15.8125 C 18.769531 14.550781 20 12.410156 20 10 C 20 6.144531 16.855469 3 13 3 Z M 13 5 C 15.773438 5 18 7.226563 18 10 C 18 12.773438 15.773438 15 13 15 C 10.226563 15 8 12.773438 8 10 C 8 7.226563 10.226563 5 13 5 Z M 26 15 C 26.253906 15 26.519531 15.082031 26.71875 15.28125 C 27.117188 15.679688 27.117188 16.289063 26.71875 16.6875 L 17.03125 26.375 L 15.25 26.75 L 15.625 24.96875 L 25.3125 15.28125 C 25.511719 15.082031 25.746094 15 26 15 Z"
              />
            </svg>
          </label>
        </div>
      </div>
      <div
        className="flex items-center justify-center text-NeutralBlack dark:text-NeutralWhite md:w-2/3 lg:w-full "
      >
        <div
          className="mx-auto w-full lg:max-w-[80%]  mb-20"
        >
          <h2
            className="lg:pt-5 pb-10 text-5xl font-semibold"
          >
            Therapist Profile
          </h2>
          <form
            className=""
          >
            <div
              className="mb-5 text-xl flex"
            >
              <label
                className=" mt-4 w-3/4 text-NeutralBlack dark:text-NeutralWhite  "
                htmlFor="name"
              >
                full name
              </label>
              <input
                className="w-full text-NeutralBlack  font-normal text-lg px-4 rounded-md border border-slate-300 bg-NeutralWhite py-3 outline-none "
                id="name"
                name="name"
                onChange={[Function]}
                type="text"
                value="mockConstructor"
              />
            </div>
            <div
              className="mb-5 text-xl flex"
            >
              <label
                className=" mt-4 w-2/4 text-NeutralBlack dark:text-NeutralWhite "
                htmlFor="Bio"
              >
                Bio
              </label>
              <textarea
                className="p-4 text-NeutralBlack  w-[68%] h-[40%] text-xl font-poppins font-normal border border-slate-300 bg-NeutralWhite outline-none lg:h-full sm:h-full sm:leading-tight rounded-md "
                id="bio"
                name="bio"
                onChange={[Function]}
                placeholder=""
                type="text"
                value=""
              />
            </div>
            <div
              className="mb-5 flex"
            >
              <label
                className=" mt-4 text-xl w-3/4 text-NeutralBlack dark:text-NeutralWhite"
                htmlFor="Birth Date"
              >
                birth date
              </label>
              <input
                className="w-full rounded-md border text-NeutralBlack  font-normal lg:text-xl px-4 border-slate-300 bg-NeutralWhite py-3 outline-none "
                name="Birth Date"
                onChange={[Function]}
                type="date"
                value=""
              />
            </div>
            <div
              className="mb-5 flex"
            >
              <label
                className=" mt-4 text-xl w-3/4 text-NeutralBlack dark:text-NeutralWhite "
                htmlFor="email"
              >
                email
              </label>
              <input
                className="w-full rounded-md border text-NeutralBlack  font-normal lg:text-xl px-4 border-slate-300 bg-NeutralWhite py-3 outline-none "
                id="email"
                name="email"
                onChange={[Function]}
                type="email"
              />
            </div>
            <div
              className="mb-5 flex"
            >
              <label
                className=" mt-4 text-xl w-3/4 text-NeutralBlack dark:text-NeutralWhite  "
                htmlFor="phone"
              >
                phone number
              </label>
              <input
                className="w-full rounded-md text-NeutralBlack  border lg:text-xl font-normal px-4 border-slate-300 bg-NeutralWhite py-3 outline-none "
                id="phone"
                name="phone"
                onChange={[Function]}
                type="number"
              />
            </div>
          </form>
          <div
            className=" py-4  flex justify-end "
          >
            <button
              className="w-28 h-10 rounded-md text-base font-poppins font-regular bg-Accent text-NeutralBlack dark:text-NeutralWhite dark:bg-Dark_Primary dark:hover:bg-[#3E4E68]  hover:bg-[#879AB8] hover:text-NeutralWhite hover:scale-105 duration-500"
              onClick={[Function]}
            >
              Update
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <footer
    className="footer mt-20 px-10 py-5 bg-Accent dark:bg-Dark_Primary text-base-content flex flex-col md:flex-row lg:flex-row justify-between items-center"
  >
    <form>
      <header
        className=" font-bold text-xl mb-[-15px] text-NeutralBlack dark:text-NeutralWhite"
      >
        footer_msg_1
      </header>
      <fieldset
        className="form-control w-96 mt-2"
      >
        <label
          className="label"
        >
          <span
            className="label-text text-NeutralBlack dark:text-NeutralWhite"
          >
            footer_msg_2
          </span>
        </label>
        <div
          className="relative flex flex-row"
        >
          <input
            className="input input-bordered border-[#718096] border-solid w-4/5 outline-none focus:outline-none"
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="footer_msg_3"
            type="text"
            value=""
          />
          <span
            className=" w-20 ml-[-1rem]"
            onClick={[Function]}
          >
            <svg
              fill="none"
              viewBox="0 0 100 100"
              width="100%"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M1 1H54C56.7614 1 59 3.23858 59 6V54C59 56.7614 56.7614 59 54 59H1V1Z"
                fill="#99B4DF"
                id="subscribe"
                stroke="#718096"
                strokeWidth="2"
              />
              <path
                d="M29.6327 30.3673L25.0421 32.5401C24.5997 32.7861 24.053 32.7438 23.6538 32.4325L17.4862 27.7498C16.6922 27.1308 16.9078 25.8769 17.8628 25.5585L40.3377 18.0669C41.3237 17.7382 42.2618 18.6763 41.9331 19.6623L34.4415 42.1372C34.1231 43.0922 32.8691 43.3078 32.2502 42.5138L27.5675 36.3462C27.2562 35.947 27.2139 35.4003 27.4599 34.9579L29.6327 30.3673Z"
                fill="#1A1A1A"
              />
            </svg>
          </span>
        </div>
      </fieldset>
    </form>
    <div
      className=" items-center flex flex-col justify-center h-full text-NeutralBlack dark:text-NeutralWhite"
    >
      <div
        className="flex flex-row items-center justify-between"
      >
        <div
          className="mb-2 px-3"
        >
          <div
            className="bg-DarkWhite dark:bg-EnglishViolet"
            onClick={[Function]}
          >
            <label
              className="toggle"
              htmlFor="switch"
            >
              <div
                className="icon iconMoon checked"
              >
                <svg
                  fill="currentColor"
                  height="20"
                  viewBox="0 0 24 24"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clipRule="evenodd"
                    d="M9.528 1.718a.75.75 0 01.162.819A8.97 8.97 0 009 6a9 9 0 009 9 8.97 8.97 0 003.463-.69.75.75 0 01.981.98 10.503 10.503 0 01-9.694 6.46c-5.799 0-10.5-4.701-10.5-10.5 0-4.368 2.667-8.112 6.46-9.694a.75.75 0 01.818.162z"
                    fillRule="evenodd"
                  />
                </svg>
              </div>
              <div
                className="icon iconSun"
              >
                <svg
                  fill="currentColor"
                  height="20"
                  viewBox="0 0 24 24"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M12 2.25a.75.75 0 01.75.75v2.25a.75.75 0 01-1.5 0V3a.75.75 0 01.75-.75zM7.5 12a4.5 4.5 0 119 0 4.5 4.5 0 01-9 0zM18.894 6.166a.75.75 0 00-1.06-1.06l-1.591 1.59a.75.75 0 101.06 1.061l1.591-1.59zM21.75 12a.75.75 0 01-.75.75h-2.25a.75.75 0 010-1.5H21a.75.75 0 01.75.75zM17.834 18.894a.75.75 0 001.06-1.06l-1.59-1.591a.75.75 0 10-1.061 1.06l1.59 1.591zM12 18a.75.75 0 01.75.75V21a.75.75 0 01-1.5 0v-2.25A.75.75 0 0112 18zM7.758 17.303a.75.75 0 00-1.061-1.06l-1.591 1.59a.75.75 0 001.06 1.061l1.591-1.59zM6 12a.75.75 0 01-.75.75H3a.75.75 0 010-1.5h2.25A.75.75 0 016 12zM6.697 7.757a.75.75 0 001.06-1.06l-1.59-1.591a.75.75 0 00-1.061 1.06l1.59 1.591z"
                  />
                </svg>
              </div>
            </label>
          </div>
        </div>
        <div
          className="mb-2.5 px-3"
        >
          <div
            className="relative inline-block text-base rounded-md"
          >
            <button
              aria-expanded={false}
              aria-haspopup="listbox"
              aria-label="Select Language"
              onClick={[Function]}
            >
              <svg
                className="inline-block w-5 h-5 mr-2"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8zm7.5-6.923c-.67.204-1.335.82-1.887 1.855A7.97 7.97 0 0 0 5.145 4H7.5V1.077zM4.09 4a9.267 9.267 0 0 1 .64-1.539 6.7 6.7 0 0 1 .597-.933A7.025 7.025 0 0 0 2.255 4H4.09zm-.582 3.5c.03-.877.138-1.718.312-2.5H1.674a6.958 6.958 0 0 0-.656 2.5h2.49zM4.847 5a12.5 12.5 0 0 0-.338 2.5H7.5V5H4.847zM8.5 5v2.5h2.99a12.495 12.495 0 0 0-.337-2.5H8.5zM4.51 8.5a12.5 12.5 0 0 0 .337 2.5H7.5V8.5H4.51zm3.99 0V11h2.653c.187-.765.306-1.608.338-2.5H8.5zM5.145 12c.138.386.295.744.468 1.068.552 1.035 1.218 1.65 1.887 1.855V12H5.145zm.182 2.472a6.696 6.696 0 0 1-.597-.933A9.268 9.268 0 0 1 4.09 12H2.255a7.024 7.024 0 0 0 3.072 2.472zM3.82 11a13.652 13.652 0 0 1-.312-2.5h-2.49c.062.89.291 1.733.656 2.5H3.82zm6.853 3.472A7.024 7.024 0 0 0 13.745 12H11.91a9.27 9.27 0 0 1-.64 1.539 6.688 6.688 0 0 1-.597.933zM8.5 12v2.923c.67-.204 1.335-.82 1.887-1.855.173-.324.33-.682.468-1.068H8.5zm3.68-1h2.146c.365-.767.594-1.61.656-2.5h-2.49a13.65 13.65 0 0 1-.312 2.5zm2.802-3.5a6.959 6.959 0 0 0-.656-2.5H12.18c.174.782.282 1.623.312 2.5h2.49zM11.27 2.461c.247.464.462.98.64 1.539h1.835a7.024 7.024 0 0 0-3.072-2.472c.218.284.418.598.597.933zM10.855 4a7.966 7.966 0 0 0-.468-1.068C9.835 1.897 9.17 1.282 8.5 1.077V4h2.355z"
                />
              </svg>
            </button>
            <div
              className="absolute bottom-10 w-36"
              style={
                Object {
                  "display": "none",
                }
              }
            >
              <ul
                className="dropdown menu py-2 bg-NeutralWhite dark:bg-Dark_Primary rounded-md "
              >
                <li>
                  <a
                    className="block px-3 py-2 text-NeutralBlack dark:text-NeutralWhite "
                    href="/en"
                    onClick={[Function]}
                  >
                    English
                  </a>
                </li>
                <li>
                  <a
                    className="block px-3 py-2 text-NeutralBlack dark:text-NeutralWhite"
                    href="/ar"
                    onClick={[Function]}
                  >
                    العربية
                  </a>
                </li>
                <li>
                  <a
                    className="block px-3 py-2 text-NeutralBlack dark:text-NeutralWhite"
                    href="/fr"
                    onClick={[Function]}
                  >
                    French
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
        <nav
          className="footer-title flex gap-8 font-[13px] md:font-medium lg:font-medium "
        >
          <a
            className=" "
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            Home
          </a>
          <a
            className=" "
            href="../blogs"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            Blogs
          </a>
          <a
            className=" "
            href="../about"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            About
          </a>
          <a
            className=" "
            href="../contact"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            Contact
          </a>
        </nav>
      </div>
      <div
        className="grid grid-flow-col gap-4"
      >
        <a
          href=""
        >
          <svg
            fill="none"
            height="35"
            viewBox="0 0 35 35"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M29.2578 4.62682C30.9347 4.28241 32.5884 3.39473 32.5884 3.39473C33.3256 3.04856 33.6012 3.31906 33.2037 4.01651C33.2037 4.01651 32.2112 6.07413 30.6949 7.02984C32.1797 6.84341 33.52 6.29924 33.52 6.29924C34.2926 6.0365 34.5007 6.66055 33.9952 7.29167C33.9952 7.29167 32.6842 9.11819 31.25 10.2083C31.2624 10.538 31.2682 10.4798 31.2682 10.8164C31.2682 22.4922 25.3761 32.8125 10.3789 32.8125C4.74279 32.8125 1.72544 30.5762 1.72544 30.5762C0.269777 29.8354 0.435023 28.9706 2.05229 28.7287C2.05229 28.7287 7.48179 28.2115 9.99381 26.1372C-4.37503 18.9583 1.48563 5.05932 1.48563 5.05932C1.54196 4.25827 2.09591 4.05424 2.68917 4.61642C2.68917 4.61642 9.15627 11.3561 16.7492 11.6789C16.6231 11.1127 16.5601 10.525 16.5601 9.91857C16.5601 5.64926 19.8479 2.1875 23.8993 2.1875C26.0129 2.1875 27.922 3.12555 29.2578 4.62682Z"
              fill="#1A1A1A"
            />
          </svg>
        </a>
        <a
          href=""
        >
          <svg
            fill="none"
            height="35"
            viewBox="0 0 35 35"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1.45837 17.5389C1.45837 8.65781 8.64047 1.45825 17.5 1.45825C26.3596 1.45825 33.5417 8.65781 33.5417 17.5389C33.5417 25.8821 27.2033 32.7413 19.0902 33.5416C19.1336 33.5339 19.1769 33.5258 19.2202 33.517V22.155H23.3908L24.0148 17.3012H19.2202V14.2023C19.2202 12.7969 19.6093 11.8395 21.6199 11.8395L24.1837 11.8382V7.49707C23.7402 7.43772 22.2185 7.30577 20.4476 7.30577C16.7511 7.30577 14.22 9.56756 14.22 13.7218V17.3012H10.0388V22.155H14.22V33.2832C6.93355 31.7655 1.45837 25.2932 1.45837 17.5389Z"
              fill="#1A1A1A"
            />
          </svg>
        </a>
        <a
          href=""
        >
          <svg
            fill="none"
            height="33"
            viewBox="0 0 33 33"
            width="33"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16.7259 13.3233V18.6751C16.7259 19.2279 17.1737 19.6759 17.7266 19.6752C19.597 19.6727 23.6064 19.6678 25.5202 19.6678C24.1419 23.787 21.9994 26.0297 16.7259 26.0297C11.3888 26.0297 7.22328 21.763 7.22328 16.4993C7.22328 11.2368 11.3888 6.97012 16.7259 6.97012C19.5473 6.97012 21.3699 7.9484 23.0415 9.3115C24.3792 7.99213 24.2675 7.80347 27.6716 4.6325C24.7816 2.03874 20.9409 0.458252 16.7259 0.458252C7.74238 0.458252 0.458374 7.6398 0.458374 16.4993C0.458374 25.3588 7.74238 32.5416 16.7259 32.5416C30.1541 32.5416 33.4363 21.0109 32.3486 13.3233H16.7259Z"
              fill="#1A1A1A"
            />
          </svg>
        </a>
      </div>
    </div>
  </footer>
</div>
`;
